.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       max-parallel: 1
       matrix:
-        goVer: ["^1.16", "^1.17"]
+        goVer: ["1.16", "1.17"]
```

Comment:
![image](https://user-images.githubusercontent.com/19969687/152314677-87555179-d078-4bee-a749-5e33c354e031.png)

Reply:
yes, it uses latest by default, previous change was unnecessary

Reply:
yup, my bad.

can you remove these checks?
![image](https://user-images.githubusercontent.com/19969687/152320478-43c8ec24-93aa-4a2c-9d5f-f2f32e2c72da.png)